samples/transform_feedback_instanced.html

```diff
@@ -0,0 +1,380 @@
+        var canvas = document.createElement('canvas');
+        canvas.width = Math.min(window.innerWidth, window.innerHeight);
+        canvas.height = canvas.width;
+        document.body.appendChild(canvas);
+
+        // -- Init WebGL Context
+        var gl = canvas.getContext('webgl2', { antialias: false });
+        var isWebGL2 = !!gl;
+        if(!isWebGL2)
+        {
+            document.getElementById('info').innerHTML = 'WebGL 2 is not available.  See <a href="https://www.khronos.org/webgl/wiki/Getting_a_WebGL_Implementation">How to get a WebGL 2 implementation</a>';
+            return;
+        }
+
+        canvas.addEventListener("webglcontextlost", function(event) {
+            event.preventDefault();
+        }, false);
+
+        // -- Init Program
+
+        var PROGRAM_TRANSFORM = 0;
+        var PROGRAM_DRAW = 1;
+
+        var programs = initPrograms();
+
+
+        // -- Initialize data
+        var NUM_INSTANCES = 1000;
+
+        var currentSourceIdx = 0;
+
+        var trianglePositions = new Float32Array([
+            0.015, 0.0, 
+            -0.010, 0.010, 
+            -0.010, -0.010,
+        ]);
+
+        var instanceOffsets = new Float32Array(NUM_INSTANCES * 2);
+        var instanceRotations = new Float32Array(NUM_INSTANCES * 1);
+        var instanceColors = new Float32Array(NUM_INSTANCES * 3);
+
+        for (var i = 0; i < NUM_INSTANCES; ++i) {
+            var oi = i * 2;
+            var ri = i;
+            var ci = i * 3;
+
+            instanceOffsets[oi] = Math.random() * 2.0 - 1.0;
+            instanceOffsets[oi + 1] = Math.random() * 2.0 - 1.0;
+
+            instanceRotations[i] = Math.random() * 2 * Math.PI;
+
+            instanceColors[ci] = Math.random();
+            instanceColors[ci + 1] = Math.random();
+            instanceColors[ci + 2] = Math.random();
+        }
+
+        // -- Init Vertex Array
+        var OFFSET_LOCATION = 0;
+        var ROTATION_LOCATION = 1;
+        var POSITION_LOCATION = 2;      // this is vertex position of the instanced geometry
+        var COLOR_LOCATION = 3;
+        var NUM_LOCATIONS = 4;
+
+        var drawTimeLocation = gl.getUniformLocation(programs[PROGRAM_DRAW], 'u_time');
+
+        var vertexArrays = [gl.createVertexArray(), gl.createVertexArray()];
+
+        var vertexBuffers = new Array(vertexArrays.length);
+
+        for (var va = 0; va < vertexArrays.length; ++va) {
+            gl.bindVertexArray(vertexArrays[va]);
+            vertexBuffers[va] = new Array(NUM_LOCATIONS);
+
+            vertexBuffers[va][OFFSET_LOCATION] = gl.createBuffer();
+            gl.bindBuffer(gl.ARRAY_BUFFER, vertexBuffers[va][OFFSET_LOCATION]);
+            gl.bufferData(gl.ARRAY_BUFFER, instanceOffsets, gl.STREAM_COPY);
+            gl.vertexAttribPointer(OFFSET_LOCATION, 2, gl.FLOAT, false, 0, 0);
+            gl.enableVertexAttribArray(OFFSET_LOCATION);
+            gl.vertexAttribDivisor(OFFSET_LOCATION, 1); // attribute used once per instance

```

Comment:
I'd remove the vertexAttribDivisor calls here for offset and rotation, since the divisor's now being updated in the render loop.